localpdb/PDB.py

```diff
@@ -212,6 +212,18 @@ def _add_col_chains(self, data, added_col_name=[]):
                                           columns=added_col_name)
         self.__chains = pd.merge(self.chains, data, left_index=True, right_index=True, how='left')
 
+    def search_seq_motif(self, query, type_='prosite'):
+        command = SearchMotifCommand(query, type_)
+        return command.execute()
+
+    def search_seq(self, sequence, evalue=1, identity=0.9):
+        command = SequenceSimilarityCommand(sequence, evalue, identity)
+        return command.execute()
+
+    def search_struct(self, pdb_id, assembly_id=1, operator = 'strict_shape_match'):
+        command = StructureSimilarityCommand(pdb_id, assembly_id, operator)
```

Comment:
Add docstring with parameters and output format.

Reply:
Thank you, added